src/inference/Inference.jl

```diff
@@ -369,6 +370,49 @@ function AbstractMCMC.bundle_samples(
     )
 end
 
+function AbstractMCMC.bundle_samples(
+    rng::AbstractRNG,
+    model::AbstractModel,
+    spl::Sampler,
+    N::Integer,
+    ts::Vector{<:AbstractTransition},
+    chain_type::Type{Vector};
+    discard_adapt::Bool=true,
+    save_state=true,
+    kwargs...
+)
+    return ts
+end
+
+function AbstractMCMC.bundle_samples(
+    rng::AbstractRNG,
+    model::AbstractModel,
+    spl::Sampler,
+    N::Integer,
+    ts::Vector{<:AbstractTransition},
+    chain_type::Type{NamedTuple};
```

Comment:
It seems more accurate would be
```suggestion
    chain_type::Type{Vector{NamedTuple}};
```
?